versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "~> 0.12.6"
```

Comment:
This is too optimistic that things won't break in 0.14 :)

Instead, can you write something like `~> 0.12.6 || ~> 0.13.0` (I didn't test this) ?

Reply:
good point. I can add an upper-bounds to allow up to but not including 0.14 (similiar to what you wrote) https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version

Reply:
I am thinking about this - `~> 0.12.6, ~> 0.13.0`. The point is that in 0.13.x may be something wrong and we may want to add another _excluding_ range after a comma.

Reply:
ok give me some time to do a quick test to see if that style works, I can't tell for sure from just reading the docs. I'll get back to you

Reply:
ok so the syntax you've proposed is not valid - the closest equivalent is what I have proposed in this PR

Reply:
Ok, good to know. Merging.